tests/python/frontend/pytorch/test_forward.py

```diff
@@ -0,0 +1,608 @@
+
+        assert_shapes_match(baseline_output, compiled_relay_output)
+        tvm.testing.assert_allclose(baseline_output, compiled_relay_output,
+                                    rtol=1e-3, atol=1e-3)
+
+    if(test_repeats > 0):
+        thresh = 1e-2
+        units = 1e3
+        thresh = int(thresh * units)
+        input_shapes = list(input_shapes.values())
+
+        compiled_latencies = []
+        baseline_latencies = []
+        speedups = []
+
+        for i in range(0, test_repeats):
+            print("computing compiled latency")
+            compiled_latency = measure_latency(relay_model, input_shapes,
+                                               output_shapes, thresh) * units
+            print(f'Compiled latency is {compiled_latency:.3f} +/- {thresh:d} ms.')
+            print("computing baseline latency")
+            baseline_latency = measure_latency(baseline_model, input_shapes,
+                                               output_shapes, thresh) * units
+
+            print(f'Baseline latency is {baseline_latency:.3f} +/- {thresh:d} ms.')
+
+            speedup = baseline_latency/compiled_latency
+            print(f'Relative speedup is {speedup:.3f}')
+
+            compiled_latencies.append(compiled_latency)
+            baseline_latencies.append(baseline_latency)
+            speedups.append(speedup)
+
+        baseline_latencies_map[model_name] = baseline_latencies
+        compiled_latencies_map[model_name] = compiled_latencies
+        speedups_map[model_name] = speedups
+        model_names.append(model_name)
+
+        print_results()
+
+    from subprocess import call
+    call('rm -rf ~/.torch/models/*', shell=True)
+
+def print_results():
+    print(baseline_latencies_map)
+    print(compiled_latencies_map)
+    print(speedups_map)
+
+    thresh = 1e-2
+    units = 1e3
+    thresh = int(thresh * units)
+
+    for model_name in model_names:
+
+        compiled_sum = 0.0
+        baseline_sum = 0.0
+        speedup_sum = 0.0
+
+        print("For model name "+model_name)
+        for i in range(0, test_repeats):
+            print(f'Compiled latency is {compiled_latencies_map[model_name][i]:.3f} +/- {thresh:d} ms.')
+            print(f'Baseline latency is {baseline_latencies_map[model_name][i]:.3f} +/- {thresh:d} ms.')
+            print(f'Relative speedup is {speedups_map[model_name][i]:.3f}')
+
+            compiled_sum = compiled_sum + compiled_latencies_map[model_name][i]
+            baseline_sum = baseline_sum + baseline_latencies_map[model_name][i]
+            speedup_sum = speedup_sum + speedups_map[model_name][i]
+
+        print(f'Average compiled latency is {compiled_sum/test_repeats:.3f} +/- {thresh:d} ms.')
+        print(f'Average baseline latency is {baseline_sum/test_repeats:.3f} +/- {thresh:d} ms.')
+        print(f'Average relative speedup is {speedup_sum/test_repeats:.3f}')
+
+# Test Functions
+def test_add1():
+    verify_model('Add1')
```

Comment:
Just had another look at the other frontend tests and I agree. Will fix.

Reply:
Currently having some issues getting the CI to pass with this change. Trying some things but the issue is that for the GPU test, we are running our of memory. Doesn't really make sense to me as this shouldn't be a functional change.

Reply:
I wonder if its because Pytorch is trying to keep around all the models being built. You might be able to create a model, run a test, then use `del model` to force its cleanup.